src/App.jsx

```diff
@@ -1,32 +1,28 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 
-import { useDispatch } from 'react-redux';
+import { Link, Switch, Route } from 'react-router-dom';
 
-import RegionsContainer from './RegionsContainer';
-import CategoriesContainer from './CategoriesContainer';
-import RestaurantsContainer from './RestaurantsContainer';
-
-import {
-  loadInitialData,
-} from './actions';
-
-// 0. 지역, 분류 목록을 얻기
-// 1. 지역 선택 - Regions <- API (0)
-// 2. 분류 선택 - Categories - 한식, 중식, 일식, ... <- API (0)
-// 3. 식당 목록 - Restaurants <- API (with region, category) -> 1, 2 모두 완료된 경우
+import HomePage from './HomePage';
+import AboutPage from './AboutPage';
+import NotFoundPage from './NotFoundPage';
+import RestaurantsPage from './RestaurantsPage';
+import RestaurantsDetailPage from './RestaurantsDetailPage';
 
 export default function App() {
-  const dispatch = useDispatch();
-
-  useEffect(() => {
-    dispatch(loadInitialData());
-  });
-
   return (
     <div>
-      <RegionsContainer />
-      <CategoriesContainer />
-      <RestaurantsContainer />
+      <header>
+        <Link to="/">
+          <h1>헤더</h1>
+        </Link>
+      </header>
+      <Switch>
+        <Route exact path="/" component={HomePage} />
+        <Route path="/about" component={AboutPage} />
+        <Route path="/restaurants/:restaurantId" component={RestaurantsDetailPage} />
```

Comment:
```suggestion
        <Route path="/restaurants/:id" component={RestaurantsDetailPage} />
```

REST에서 URL을 표현할 땐 보통 id라고 얘기합니다. 컬렉션인 restaurants에서 식별자(id)로 구분한다는 사실이 중요하죠.

Reply:
ㅎㅎ 변수명은 항상 독립적으로 생각해야하네요. 감사합니다.